db/create-db.sql

```diff
@@ -71,30 +71,34 @@ CREATE TABLE emissions(
 -- ------------------------------------------------------------------------------------
 
 -- Have started to populate but need to get better carbon emissions figures.
+-- LJ Notes :
+-- The API returns driving, walking, bicycling and transit. 
+-- Transit would take into consideration just bus and private vehicle can take into consideration a diesel car or an elec vehicle.
+-- Private vehicle can take into consideration if travelling with multiple people. Emission sharing i.e Emission/no.of people travelling
```

Comment:
Also agree. Perhaps this could be a "good to have" and we sort this after we have a working prototype.